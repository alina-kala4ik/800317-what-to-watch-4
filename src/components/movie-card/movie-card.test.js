import React from "react";
import renderer from "react-test-renderer";
import MoveCard from "./movie-card.jsx";

const film = {
  title: `Fantastic Beasts: The Crimes of Grindelwald`,
  src: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
};

it(`render MoveCard`, () => {
  const tree = renderer
    .create(<MoveCard
      film={film}
      onCardActive={()=>{}}
      onFilmTitleClick={()=>{}}
    />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
