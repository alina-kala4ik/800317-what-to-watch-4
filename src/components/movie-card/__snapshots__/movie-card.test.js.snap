// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render MoveCard 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <div
    className="small-movie-card__image"
    onClick={[Function]}
  >
    <video
      autoPlay={false}
      height="175"
      loop={true}
      poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
      preload="none"
      src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
      onClick={[Function]}
    >
      Fantastic Beasts: The Crimes of Grindelwald
    </a>
  </h3>
</article>
`;
